.circleci/config.yml

```
@@ -1,92 +1,36 @@
 version: 2.1
 
+setup: true
 orbs:
-  path-filtering: circleci/path-filtering@<<pipeline.parameters.dev-orb-version>>
-  orb-tools: circleci/orb-tools@10.0
-  shellcheck: circleci/shellcheck@2.0
+  orb-tools: circleci/orb-tools@11.4
+  shellcheck: circleci/shellcheck@3.1
 
-# Pipeline Parameters
-## These parameters are used internally by orb-tools. Skip to the Jobs section.
-parameters:
-  run-integration-tests:
-    description: An internal flag to prevent integration test from running before a development version has been created.
-    type: boolean
-    default: false
-  dev-orb-version:
-    description: >
-      The development version of the orb to test.
-      This value is automatically adjusted by the "trigger-integration-tests-workflow" job to correspond with the specific version created by the commit and should not be edited.
-      A "dev:alpha" version must exist for the initial pipeline run.
-    type: string
-    default: "dev:alpha"
-
-jobs:
-  # Define one or more jobs which will utilize your orb's commands and parameters to validate your changes.
-  integration-test-1:
-    docker:
-      - image: cimg/base:stable
-    steps:
-      - checkout
-      # TODO This.
+filters: &filters
+  tags:
+    only: /.*/
```

Shouldn't this be
```suggestion
    only: /.+/
```
so that we have at least one character in the tag?

(Same in the other config.)